doc-utils/DevExample.tsx

```diff
@@ -7,11 +7,11 @@ interface Props extends ComponentExampleProps {
     scrollable?: boolean;
 }
 
-export const DevExample: FC<Props> = ({ title, knobs, component, scrollable }) => {
+export const DevExample: FC<Props> = (props) => {
```

Comment:
Trenger denne lenger å være en separat komponent, nå som den deler omtrent alt med `ComponentExample`?

Reply:
Den har bittelitt custom CSS for å gjøre opplevelsen bedre utenfor kontekst av portalen, så jeg ville ikke fjerne den helt